### PR TITLE
Finish up work on Edit Your Property Backend

### PR DESCRIPTION
**Pull request by AndySana**
I finished up the Java code for the edit function in view my list.

### CODE REVIEW

File: app/src/main/java/com/example/lawn_care/editYourProperty.java [hunk 1]
```diff
@@ -89,17 +104,410 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        else
+            CB_irrigation.setChecked(false);
+        if (workNeededEdit.contains("Aeration") == true)
+            CB_aeration.setChecked(true);
+        else
+            CB_aeration.setChecked(false);
+
     }
 
     public void ResetAll(View view) {
```
Comment (mjyousif) [inline app/src/main/java/com/example/lawn_care/editYourProperty.java]: what does this function do? shouldn't it just reset

Comment (SiddPpatel) [reply app/src/main/java/com/example/lawn_care/editYourProperty.java]: Yes its's just reset all the fields with Spinner

File: app/src/main/java/com/example/lawn_care/editYourProperty.java [hunk 2]
```diff
@@ -89,17 +104,410 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        spinnerState.setSelection(0,true);
-        ET_address.setText("");
-        ET_city.setText("");
-        ET_zipcode.setText("");
-        ET_PropertySize.setText("");
+        switch(stateEdit)
```
Comment (SiddPpatel) [inline app/src/main/java/com/example/lawn_care/editYourProperty.java]: You can just do spinnerState.getText() and verify the text from spinner

Comment (edgarefigueroa) [inline app/src/main/java/com/example/lawn_care/editYourProperty.java]: "switch(stateEdit)" twice? 109 and 312

**Pull request merged**